sdcm/test_config.py

```diff
@@ -252,6 +252,8 @@ def get_startup_script(cls) -> str:
             syslog_host_port=host_port,
             logs_transport=cls._tester_obj.params.get('logs_transport') if cls._tester_obj else "rsyslog",
             disable_ssh_while_running=True,
+            configure_sshd=True,
+            legacy_rsa_keys=True
```

Comment:
again, would this be called to all tests? or should be done for Ubuntu22 only?

Reply:
IMHO this can be called for all tests.